### PR TITLE
333 create a countdown bossbar for events

### PR DESCRIPTION
**Pull request by Makkuusen**

### CODE REVIEW

**Comment by Makkuusen** [comment]
Now we can select an event and do /event countdown start 5m <label>

File: src/main/java/me/makkuusen/timing/system/event/EventCountdown.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+        event.getSpectators().values().forEach(spectator -> {
+            Player player = spectator.getTPlayer().getPlayer();
+            if (player != null) {
+                player.showBossBar(bossBar);
+            }
+        });
+    }
+
+    public boolean isActive() {
```
Comment (Pigalala) [inline src/main/java/me/makkuusen/timing/system/event/EventCountdown.java:109]: Can't this getter be handled by lombok? Not a big thing but it's a bit triggering for me :pensive:

Comment (Makkuusen) [reply src/main/java/me/makkuusen/timing/system/event/EventCountdown.java:109]: yeah, I can tidy it up a bit.

**Comment by Pigalala** [comment]
Haven't tested it personally, but I don't see a problem with this system for now

**Pull request merged**